php/php.editor/src/org/netbeans/modules/php/editor/completion/DocRenderer.java

```diff
@@ -454,6 +454,20 @@ private void extractPHPDocBlock() {
                     others.toString()));
         }
 
+        private boolean tagInTagsList(List<PHPDocTag> tags, AnnotationParsedLine tagKind) {
```

Comment:
Likely not a problem here, but using a `Set` and simple lookup instead of `List` and iteration would be (much) better here. Would that be possible?

Reply:
For the reason that I am a php developer, I know java only superficially :) Thanks for the tip. I will keep it in mind next time.

Reply:
OK, so maybe @junichi11 could try to improve it one day :)